examples/plotting/file/slider.py

```diff
@@ -14,19 +14,19 @@
 plot = figure(y_range=(-10, 10), plot_width=400, plot_height=400)
 plot.line('x', 'y', source=source, line_width=3, line_alpha=0.6)
 
-callback = CustomJS(args=dict(source=source), code="""
-    var data = source.get('data');
-    var A = amp.get('value')
-    var k = freq.get('value')
-    var phi = phase.get('value')
-    var B = offset.get('value')
-    x = data['x']
-    y = data['y']
-    for (i = 0; i < x.length; i++) {
-        y[i] = B + A*Math.sin(k*x[i]+phi);
-    }
-    source.trigger('change');
-""")
+
+@CustomJS
+def callback(source=source):
+    data = source.get('data')
+    A = amp.get('value')
+    k = freq.get('value')
+    phi = phase.get('value')
+    B = offset.get('value')
+    x, y = data['x'], data['y']
+    for i in range(len(x)):
+        y[i] = B + A*Math.sin(k*x[i]+phi)
+    source.trigger('change') 
+
```

Comment:
Is there any way we could make this API match the Python API for HasProps/PlotObject? It looks like the main thing would be to change `source.data` in Python to `source.get('data')` in JS.